db/migrate/20221117065955_change_column_name.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeColumnName < ActiveRecord::Migration[6.1]
+  def change
+    rename_column :options, :option1, :option
```

Comment:
change this filed to some other name because the table name and field name shouldn't be the same